packages/help-center/src/components/help-center.tsx

```diff
@@ -77,10 +70,7 @@ const HelpCenter: React.FC< Container > = ( { handleClose } ) => {
 		};
 	}, [ portalParent ] );
 
-	return createPortal(
-		<HelpCenterContainer handleClose={ handleClose } isLoading={ isLoading } />,
-		portalParent
-	);
```

Comment:
I removed the loading blockage because it loads for nothing when the help center lands in the chat after a navigation. I moved the loading blockage to the "Still need help?" button.